### PR TITLE
housekeeping

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -5,8 +5,8 @@ lazy val scala212 = "2.12.8"
 ThisBuild / headerLicense  := Some(HeaderLicense.Custom(
   s"""Compiler Interface
      |
-     |Copyright 2011 - 2019, Lightbend, Inc.
-     |Copyright 2008 - 2010, Mark Harrah
+     |Copyright 2011-2020, Lightbend, Inc.
+     |Copyright 2008-2010, Mark Harrah
```
Comment (eed3si9n) [inline build.sbt]:
```suggestion
     |Copyright Mark Harrah and Lightbend, Inc.
```

Maybe we can drop the year range, like we did with scala/scala.

Comment (SethTisue) [reply build.sbt]: good suggestion, will do

**Comment by SethTisue** [comment]
Travis-CI likes it now.

**Review by eed3si9n — APPROVED** [review]
LGTM

**Pull request merged**